crates/orderbook/src/ipfs_app_data.rs

```diff
@@ -13,11 +13,14 @@ pub struct IpfsAppData {
 }
 
 #[derive(prometheus_metric_storage::MetricStorage, Clone, Debug)]
-#[metric(subsystem = "api")]
+#[metric(subsystem = "ipfs")]
 struct Metrics {
     /// Number of completed IPFS app data fetches.
     #[metric(labels("outcome", "source"))]
-    ipfs_app_data: prometheus::IntCounterVec,
+    app_data: prometheus::IntCounterVec,
+
+    /// Timing of IPFS app data fetches.
+    fetches: prometheus::Histogram,
```

Comment:
Should this new metric also be initialized on start up to avoid broken graphs when no data has been generated?

Reply:
Its a histogram, so I don't think its needed. AFAIK, we don't do it for our other histograms.